corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -297,7 +297,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
 
         iconKeyAction: function (e) {
-            if (e.keyCode === 13) {
+            if (e.keyCode === 13 || e.keyCode === 32) {
+                if (e.keyCode === 32) {
+                    e.preventDefault();
+                }
```

Comment:
This commit is unrelated to alt text but is accessibility-related and explicitly called out in making a "button" keyboard accessible. Ideally it would have been part of https://github.com/dimagi/commcare-hq/pull/33914 but I just now read the guideline and it seemed small enough to include here.